s3psetup/clean_open_ldap_by_s3.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
copyright header.

I guess this script is going to be temporary.. Please add this in comment.

We can have name of the file as : s3openldap_config_cleanup.py !

Reply:
Considering this is a temporary script, can we let it be a shell script for the time being? Converting it to python is additional work and I feel we may not need to spend time on this conversion as the base script is already written, working and tested. And this script is also temporary.

Reply:
keep it shell script, but add comment that this is temp file.